modules/nf-core/semibin/singleeasybin/main.nf

```diff
@@ -0,0 +1,45 @@
+process SEMIBIN_SINGLEEASYBIN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::semibin=1.4.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/semibin%3A1.4.0--pyh7cba7a3_0':
+        'quay.io/biocontainers/semibin%3A1.4.0--pyh7cba7a3_0' }"
```

Comment:
Are these correct? Your tests are failing because it can't find these containers. (possibly because of `%3A` in the name)

Reply:
`docker: invalid reference format: repository name must be lowercase.`

Reply:
In the local testing, I have tested with singularity, it works.

Reply:
It needs to pass the testing in all three ways?

Reply:
It's preffered that all tests pass, but the once for docker and singularity are really necessary to be sure it works almost everywhere